README.md

```diff
@@ -55,28 +55,36 @@ Zebra aims to be
 [faster, more secure, and more easily extensible](https://doc.zebra.zfnd.org/zebrad/index.html#zebra-advantages)
 than other Zcash implementations.
 
-## Beta Releases
+## Release Candidates
 
-Every few weeks, we release a new Zebra beta [release](https://github.com/ZcashFoundation/zebra/releases).
+Every few weeks, we release a new Zebra [release candidate](https://github.com/ZcashFoundation/zebra/releases).
```

Comment:
I think we still need to discuss and decide what we're going to call releases after the "audit" release candidate and how we're going to deal with ongoing development once we have tagged this. 

For the purpose of this document and to not block things further, I would just say "we release a new Zebra version"